src/org/exist/start/start.config

```diff
@@ -104,10 +104,4 @@ tools/jetty/lib/*                                  mode == standalone
 tools/jetty/lib/*                                  mode == other
 tools/ircbot/lib/*                                 mode == jetty
 tools/aspectj/lib/*                                always
-tools/yajsw/wrapperApp.jar                         mode == other
-tools/yajsw/wrapper.jar                            mode == other
-tools/yajsw/lib/extended/velocity/velocity-%latest%.jar         mode == other
-tools/yajsw/lib/core/commons/commons-configuration2-%latest%.jar mode == other
-tools/yajsw/lib/core/commons/commons-lang3-%latest%.jar          mode == other
-tools/yajsw/lib/core/netty/netty-all-4.0.35.Final.jar           mode == other
-tools/yajsw/lib/core/jna/jna-platform-%latest%.jar              mode == other
+tools/yajsw/lib/core/commons/commons-configuration2-%latest%.jar mode == other
```

Comment:
I don't understand where the problem is: I only removed some jar dependencies for one launch mode, not added a new one. commons-configuration2 was already referenced.